templates/ext/rbs_extension/constants.c.erb

```diff
@@ -0,0 +1,179 @@
+#include "rbs_extension.h"
+
+VALUE RBS_Parser;
+
+VALUE RBS;
+VALUE RBS_AST;
+VALUE RBS_AST_Declarations;
+VALUE RBS_AST_Directives;
+VALUE RBS_AST_Members;
+VALUE RBS_Parser;
+VALUE RBS_Types;
+VALUE RBS_Types_Bases;
+
+<%-
```

Comment:
Should we move this to `template.rb`?

Reply:
They're different per file, so it it's easier to just put them within the files they pertain to